packages/backend/Dockerfile

```diff
@@ -9,7 +9,7 @@
 #
 # Once the commands have been run, you can build the image using `yarn build-image`
 
-FROM node:18-bullseye-slim
+FROM node:18-bookworm-slim
```

Comment:
For this file you'll need to make a few more changes to resolve `error: externally-managed-environment` in the builds. I would replace lines 18 to 23 with this:

```docker
RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
    --mount=type=cache,target=/var/lib/apt,sharing=locked \
    apt-get update && \
    apt-get install -y --no-install-recommends libsqlite3-dev python3 python3-pip python3-venv build-essential && \
    yarn config set python /usr/bin/python3

ENV VIRTUAL_ENV=/opt/venv
RUN python3 -m venv $VIRTUAL_ENV
ENV PATH="$VIRTUAL_ENV/bin:$PATH"

RUN pip3 install mkdocs-techdocs-core==1.1.7
```

Reply:
Think I will separate libsqlite3-dev package to a different run as it is not needed when running external DB (From what I understand)

Reply:
Updated the Dockerfile